examples/06_default_doctrine_prepared_query_console.php

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env php
+<?php
+
+require __DIR__.'/../vendor/autoload.php';
+
+use Doctrine\DBAL\Connection;
+use Extraload\Extractor\Doctrine\PreparedQueryExtractor;
+use Extraload\Loader\ConsoleLoader;
+use Extraload\Pipeline\DefaultPipeline;
+use Extraload\Transformer\NoopTransformer;
+use Symfony\Component\Console\Helper\Table;
+use Symfony\Component\Console\Output\ConsoleOutput;
+
+$config = new \Doctrine\DBAL\Configuration();
+$connectionParams = array(
```

Comment:
You can share `$connectionParams` between your examples, it will be easier to configure only once and be able to run all examples that way.

Reply:
Yeah, let's do this.